kod/object/active/flag.kod

```diff
@@ -695,8 +695,7 @@ messages:
       return poClaimer;
    }
 
-   SetFaction(faction=FACTION_NEUTRAL)
-   "for admin use"
+   SetFaction(faction=FACTION_NEUTRAL,claim=TRUE)
```

Comment:
The new parameter seems good, but the name isn't super clear.  I'd say either come up with a name that is self-explanatory, or describe what it does in the function comment.

Also, I'm guessing there is an existing call of SetFaction somewhere in the territory game,*not* in Recreate(), that doesn't set "claim" and thus gets TRUE?  Maybe change that call to explicitly set TRUE so it's obvious that you mean to set TRUE, rather than maybe you forgot to update it?  (Default arguments are dangerous that way.)

Reply:
On second (third?) thought, `SetFaction` is only used in batch processing loops of `PlaceFlagsPartialGame`, `PlaceFlagsFullGame`, `ReloadSavedFlagStates`, `RecalcTownFlags`, and `UpdateFlagsPartialGame`. Unless there's a practical need for DMs to set the faction and update the territory game, perhaps `claim` and the the call to `FlagClaimed` should be removed?

If there is an admin need, maybe `saveGame` is a more appropriate param name?